.github/workflows/update-conformance-test-reports.yml

```diff
@@ -37,8 +37,8 @@ jobs:
           path: reference-checker
       - name: Create or update the Reference Checker ${{ env.branch }} branch
         run: |
-          git config user.name 'update-conformance-test-reports[bot]'
-          git config user.email 'update-conformance-test-reports[bot]@users.noreply.github.com'
+          git config user.name 'jspecify-bot'
```

Comment:
[Dependabot uses "dependabot[bot]."](https://github.com/google/truth/commits?author=dependabot%5Bbot%5D) I wonder if that suffix is some kind of preferred pattern?